### DISCUSSION TITLE
Add json response to make_offer

### DISCUSSION THREAD

**Discussion by xchChiaFarmers** [Chia-Network/chia-blockchain#16274] [Ideas]
Upon calling `chia wallet make_offer -f 123 -o 1:1 -r 2:1000 -p offers/test.offer`, there is no structures response. The user only gets an unstructured response in the terminal:
```
Created offer with ID 8b734...
Use chia wallet get_offers --id 8b734... -f 202463842 to view status
```

The current work arounds to get data are extremely bulky. I'd wish to see a json response with:
- offer id
- offer key
- status
- created at
- confirmed at
- accepted at
- success
- ...

Retrieving some information from an Offer file by calling `cdv inspect spendbundles <offer_file>` is extremely bulky.

BR ChiaFarmers

**Comment by BrandtH22** [comment]
Hey, I know we have discussed this in discord but for visibility I am adding it here also.

Instead of using the CLI command you may find it more applicable to use the RPC which will return a json blob with the information you have listed:
https://docs.chia.net/offer-rpc#get_offer

Will this be sufficient for what you are needing?

**Reply by xchChiaFarmers** [reply]
Thanks, that would work indeed!

I was able to find all offer-related commands also as RPC.
